components/MeetupCard.vue

```diff
@@ -1,45 +1,74 @@
                     logoBg ? logoBg : null
                 }`"
                 :src="logo"
                 :alt="`${name}-logo`"
             />
-            <h3 class="ml-6 text-xl font-medium pr-8">{{ name }}</h3>
-        </div>
-        <div class="mt-2">
-            <div class="text-grey-700">
-                <component
-                    :is="linkTo ? 'a' : 'div'"
-                    :href="linkTo || ''"
-                    class="focus:outline-none"
+            <h3 class="text-xl font-medium pr-8">{{ name }}</h3>
+            <span
+                v-if="linkTo"
+                class="pointer-events-none absolute top-0 right-0 text-gray-300 group-hover:text-brand-primary"
+                aria-hidden="true"
+            >
+                <svg
+                    class="h-6 w-6"
+                    xmlns="http://www.w3.org/2000/svg"
+                    fill="currentColor"
+                    viewBox="0 0 24 24"
                 >
-                    <!-- Extend touch target to entire panel -->
-                    <div class="flex items-center">
-                        <div class="absolute inset-0" aria-hidden="true"></div>
-                        <div v-if="slack" class="flex items-center">
-                            <svg
-                                class="w-5 h-5 mr-2"
-                                fill="#868e96"
-                                viewBox="0 0 24 24"
-                                aria-hidden="true"
-                                preserveAspectRatio="xMidYMid meet"
-                            >
-                                <path
-                                    d="M6 15a2 2 0 0 1-2 2a2 2 0 0 1-2-2a2 2 0 0 1 2-2h2v2m1 0a2 2 0 0 1 2-2a2 2 0 0 1 2 2v5a2 2 0 0 1-2 2a2 2 0 0 1-2-2v-5m2-8a2 2 0 0 1-2-2a2 2 0 0 1 2-2a2 2 0 0 1 2 2v2H9m0 1a2 2 0 0 1 2 2a2 2 0 0 1-2 2H4a2 2 0 0 1-2-2a2 2 0 0 1 2-2h5m8 2a2 2 0 0 1 2-2a2 2 0 0 1 2 2a2 2 0 0 1-2 2h-2v-2m-1 0a2 2 0 0 1-2 2a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2a2 2 0 0 1 2 2v5m-2 8a2 2 0 0 1 2 2a2 2 0 0 1-2 2a2 2 0 0 1-2-2v-2h2m0-1a2 2 0 0 1-2-2a2 2 0 0 1 2-2h5a2 2 0 0 1 2 2a2 2 0 0 1-2 2h-5z"
-                                />
-                            </svg>
-                            {{ slack }} on slack
-                        </div>
-                    </div>
-                </component>
-            </div>
-            <div v-if="durationInfo" class="flex items-center text-grey-700">
+                    <path
+                        d="M20 4h1a1 1 0 00-1-1v1zm-1 12a1 1 0 102 0h-2zM8 3a1 1 0 000 2V3zM3.293 19.293a1 1 0 101.414 1.414l-1.414-1.414zM19 4v12h2V4h-2zm1-1H8v2h12V3zm-.707.293l-16 16 1.414 1.414 16-16-1.414-1.414z"
+                    />
+                </svg>
+            </span>
+            <component
+                :is="linkTo ? 'a' : 'div'"
+                :href="linkTo || ''"
+                class="focus:outline-none"
+            >
+                <!-- Extend touch target to entire header -->
```

Comment:
This was another trade off that came from the StartupTNT card. When the full card was clickable we couldn't add any other links to the body. 

The issue requested a link to the join the discord to I tried moving the main link to the title. Open to suggestions though.

Reply:
Oh blah, yeah I see. 

Have you been to a tnt recently? are they even still on discord?

I can't find the discord link from their website.....

I don't know that I want to make all the meeting cards have less clicking just for their discord 😂

Reply:
They're definitely still going on. It sounds like they've moved back to in person as I was on a call last week with someone who's gone recently. Sounded like the last month.

Zack Storms was also at some of the Startup Edmonton Week events.

Reply:
Maybe we can just not include the discord then? if they are in person?

Reply:
OK yeah let's remove it.

I think as the resources section grows it will likely get moved into its own page, maybe at that point we can look as displaying more information with additional links.

Reply:
Reverted in 631c0fcea8c34f6fa5ce3a120157ccc35be07066 since not adding the Discord link.